### PR TITLE
Add `Online Help` link to gRPC interface's `Platform`

### PR DESCRIPTION
**Pull request by MatteoPologruto**
## Please check if the PR fulfills these requirements

See [how to contribute](https://arduino.github.io/arduino-cli/latest/CONTRIBUTING/)

- [x] The PR has no duplicates (please search among the [Pull Requests](https://github.com/arduino/arduino-cli/pulls)
      before creating one)
- [x] The PR follows
      [our contributing guidelines](https://arduino.github.io/arduino-cli/latest/CONTRIBUTING/#pull-requests)
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] `UPGRADING.md` has been updated with a migration guide (for breaking changes)

## What kind of change does this PR introduce?
Code enhancement

## What is the current behavior?
The `Online Help` link present in the [package-index.json](https://downloads.arduino.cc/packages/package_index.json) cannot be retrieved.

## What is the new behavior?
It is possible to have access to `Online Help` using `core search --format json`.
 ```
PS C:\Users\m.pologruto\Desktop\Arduino\arduino-cli> .\arduino-cli core search avr --format json
[
...
 {
    "id": "littleBits:avr",
    "latest": "1.0.0",
    "name": "littleBits Arduino AVR Modules",
    "maintainer": "littleBits Electronics",
    "website": "http://www.littlebits.cc/bits/w6-arduino",
    "email": "support@littlebits.cc",
    "boards": [
      {
        "name": "littleBits w6 Arduino module"
      }
    ],
    "type": [
      "Arduino@Heart"
    ],
    "help": {
      "online": "http://www.littlebits.cc/arduino-help"
    }
  }
]
```

## Does this PR introduce a breaking change, and is [titled accordingly](https://arduino.github.io/arduino-cli/latest/CONTRIBUTING/#breaking)?
No

### CODE REVIEW

File: rpc/cc/arduino/cli/commands/v1/common.proto [hunk 1]
```diff
@@ -126,3 +129,9 @@ message Profile {
   // FQBN specified in the profile.
   string fqbn = 2;
 }
+
+message Online {
+  // A URL provided by the author of the platform's package, intended to point
+  // to their online help service.
+  string online = 1;
+}
```
Comment (cmaglie) [inline rpc/cc/arduino/cli/commands/v1/common.proto:137]: It sounds a bit weird to call this message `Online`, what about a more self-explanatory name like `Help` or better `HelpResources`?

Comment (cmaglie) [reply rpc/cc/arduino/cli/commands/v1/common.proto:137]: I'm suggesting to rename the structure but not the field, for example:

```
message HelpResources {
  // A URL provided by the author of the platform's package, intended to point
  // to their online help service.
  string online = 1;
}
```

This "message" has the same `Online` field that you can access via `obj.Help.Online`.

Comment (MatteoPologruto) [reply rpc/cc/arduino/cli/commands/v1/common.proto:137]: Oh, I misunderstood what you meant. I'll modify it as suggested.

**Review by per1234 — APPROVED** [review]
Thanks Matteo!

**Review by umbynos — APPROVED** [review]
Code LGTM

**Pull request merged**